drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/ScenarioSimulationContext.java

```diff
@@ -251,6 +251,14 @@ public ScenarioExpressionCellTextAreaSingletonDOMElementFactory getScenarioExpre
         return getAbstractScesimGridModelByGridWidget(gridWidget).getScenarioExpressionCellTextAreaSingletonDOMElementFactory();
     }
 
+    public void setUndoButtonEnabledStatus(boolean enabled) {
```

Comment:
Why these setters?

Reply:
@danielezonca Because of this https://github.com/kiegroup/drools-wb/pull/1390/files#diff-62f87d136d61c90564bc41cf80af7a36L134 . I don't like direct field access :)